packages/blockchain-link/src/workers/solana/tokenUtils.ts

```diff
@@ -0,0 +1,36 @@
+import { TokenInfo } from '../..';
+import { transformTokenInfo } from '@trezor/blockchain-link-utils/lib/solana';
+
+const TOKEN_WHITELIST = [
+    {
+        mint: 'So11111111111111111111111111111111111111112',
+        name: 'Wrapped SOL',
+        symbol: 'WSOL',
+    },
+    {
+        mint: 'So11111111111111111111111111111111111111112',
+        name: 'Tether',
+        symbol: 'USDT',
+    },
+    {
+        mint: '4k3Dyjzvzp8eMZWUXbBCjEvwSkkk59S5iCNLY3QrkX6R',
+        name: 'Raydium',
+        symbol: 'RAY',
+    },
+];
+const fetchSolanaTokenList = async (): Promise<
+    Array<{ mint: string; name: string; symbol: string }>
+> =>
+    Promise.resolve(TOKEN_WHITELIST);
+
+export const parseTokenInfo = async (
```

Comment:
I would remove tihs fn and do the logic in `/solana/index`, the name of this function is `parse` even tho it also fetches.  I would keep the fetching in the `index` and pass stuff to `transformTokens`